cpu/saml21/periph/timer.c

```diff
@@ -48,6 +48,10 @@ static inline void _irq_enable(tim_t dev);
  */
 int timer_init(tim_t dev, unsigned long freq, timer_cb_t cb, void *arg)
 {
+    /* at the moment, the timer can only run at 1MHz */
```

Comment:
This is not related to -Wall -Wextra, or is it ?

Reply:
> This is not related to -Wall -Wextra, or is it ?

Ping @smlng?

Reply:
mhm, IIRC copy pasted from `samd21/periph/timer.c`, but don't know why/how it slipped in here. Shall I remove?

Reply:
I don't care it can stay.